CONTRIBUTING.adoc

```diff
@@ -0,0 +1,29 @@
+Whenever you want to commit a change to the skupper-router repo, please first create a PR so it can be reviewed and approved by others.
+When creating a PR, please use the following commit message pattern
+
+```
+Fixed #<github-issue-number>: <Commit message>
+```
+
+For example,
+
+```
+Fixed #90: Fixed code in http2 adaptor that was doing improper q2 related logging
+```
+
+In the above commit message, `#90` refers to the github issue - https://github.com/skupperproject/skupper-router/issues/90 (you will need to create an issue if there isn't already one)
+
+After the PR is approved, please use the Github PR "Squash and merge" button  to commit your changes to the main branch.
+Please refrain from directly committing your work to the main branch.
+The following things happen when using the "Squash and merge" button to commit your changes:
+
+1. The PR is automatically closed
+2. The github issue `#90` is closed
+3. The commit on the main branch is referenced in issue `#90`
+4. There is a hyperlink to the Github issue `#90` from the commit.
+
+For porting upstream qpid-dispatch issues to skupper-router you might use the standard `DISPATCH-XXXX: <Commit message>`  in your commit message but make sure you commit your change via the Github "Squash and merge" button.
+
+Please pull up any open PR on the skupper-router (for example - https://github.com/skupperproject/skupper-router/pull/80 ) and make sure you see the "Squash and merge" button.
```

Comment:
Can we just remove the link https://github.com/skupperproject/skupper-router/pull/80. Just say "Please pull up any open PR on the skupper-router and make sure you see the "Squash and merge" button.

Reply:
done